content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,231 @@
+============
+Product type
+============
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+Define *product types* in Odoo to track products in varying levels of detail.
+
+Classify products as *storable* to track stock counts, allowing users to trigger :doc:`reordering
+rules <../product_replenishment/reordering_rules>` for generating purchase orders. *Consumable*
+products are assumed to always be in stock, and *service* products are performed and served by the
+business.
+
+.. seealso::
+   `Odoo Tutorials: Product Type <https://www.youtube.com/watch?v=l6j0ZkP5mLM>`_
+
+Set product type
+================
+
+To set a product type, go to :menuselection:`Inventory app --> Products --> Product`, and select
+the desired product.
+
+On the product form, in the :guilabel:`Product Type` field, select:
+
+- :guilabel:`Storable Product` for products tracked with stock counts. Only storable products can
+  trigger reordering rules for generating purchase orders
+
+  .. tip::
+    Choose :guilabel:`Storable Product` if it is necessary to track a product's stock at various
+    locations, inventory valuations, or if the product has lots and/or serial numbers.
+
+- :guilabel:`Consumable`: products that are always assumed to be in stock, but with untracked and
+  unforecasted quantities (e.g. nails, toilet paper, coffee, etc.). Consumables are replaceable and
+  essential, but exact counts are unnecessary
+- :guilabel:`Service`: perform and sell services (e.g. maintenance, installation, repair)
+
+  .. image:: type/product-form.png
+    :align: center
+    :alt: Set a product type on the product form.
+
+.. note::
+   The product types listed above are part of the standard *Inventory* app. For access to the fields
+   below, :ref:`install <general/install>` the corresponding apps **in addition** to *Inventory*.
```

Comment:
The ref is not formatting in the HTML here. Did you mean to link to [this doc](https://www.odoo.com/documentation/17.0/applications/general/apps_modules.html)?

Reply:
I'm not sure what you mean by "not formatting in the HTML".

This is the rendered result I see:
![image](https://github.com/odoo/documentation/assets/23611711/2700078f-c5ae-46aa-ae3f-70bac27ec634)

And when I click on the link, I go to:
![image](https://github.com/odoo/documentation/assets/23611711/739ff09f-479a-405a-8290-9162c5e12b49)

I do intend to link to the doc you listed, but specifically, the "Install apps and modules" section

![image](https://github.com/odoo/documentation/assets/23611711/8676e0ea-08eb-43b4-9868-68ad7ba2ee06)